tools/check-enforcer/Azure.Sdk.Tools.CheckEnforcer/Handlers/PullRequestHandler.cs

```diff
@@ -36,7 +36,7 @@ protected override async Task HandleCoreAsync(HandlerContext<PullRequestEventPay
                 runIdentifier
                 );
 
-            if (action == "opened" || action == "reopened")
+            if (action == "opened" || action == "reopened" || action == "synchronize")
```

Comment:
What is the synchronize action?

Reply:
It happens when a PR gets a new commit. I used telemetry to figure out the event actions that get fired when I push to a branch omcivered by a PR.